src/renderer/shared/util/getStorage.js

```diff
@@ -0,0 +1,25 @@
+import { wallet, api, rpc, u } from '@cityofzion/neon-js';
+
+export const encode = (value) => {
```

Comment:
we could write this like so:
```
export const encode = value => u.str2hexstring(value);
```
Having it like this is an implicit return statement - also It's shorter and cleaner xD
(I think our linter will add parentheses around `value`, but that's okay)